src/index.html

```diff
@@ -1,13 +1,92 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html
+  lang="en"
+  class="html"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link
+      rel="preload"
+      href="fonts/Roboto-Bold.woff"
+      as="font"
+      type="font/woff"
+      crossorigin="anonymous"
+    >
+    <link
+      rel="preload"
+      href="fonts/Roboto-Medium.woff"
+      as="font"
+      type="font/woff"
+      crossorigin="anonymous"
+    >
+    <link
+      rel="preload"
+      href="fonts/Roboto-Regular.woff"
+      as="font"
+      type="font/woff"
+      crossorigin="anonymous"
+    >
+    <link
+      rel="stylesheet"
+      href="styles/style.css"
+    >
+    <title>Product cards</title>
+  </head>
+  <body class="body">
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac Retina Monoblock"
+        class="card__image"
+      >
+
+      <h3 class="card__heading">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="card__rate">
+        <div
+          class="card__stars stars stars--4"
+          aria-label="Rating of this product is 4 out of 5."
+        >
+          <span class="stars__star"></span>
+
+          <span class="stars__star"></span>
+
+          <span class="stars__star"></span>
+
+          <span class="stars__star"></span>
+
+          <span class="stars__star"> </span>
+        </div>
+
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
```

Comment:
```suggestion
        <div
          class="card__stars stars stars--4"
          aria-label="Rating of this product is 4 out of 5."
        >
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"> </span>
        </div>
        <p class="card__reviews">Reviews: 5</p>
      </div>
```

Reply:
Use blank lines to separate big blocks of code, not for every element

Reply:
Ok, will change this soon